.travis.yml

```diff
@@ -20,7 +20,7 @@ env:
     - SESSION_RESET_THRESHOLD_TEST=2000
```

Comment:
Friendly reminder to add a line in the CHANGELOG explaining what changed.

Reply:
Thank you, will do! : )